src/middleware/request-logger.middleware.ts

```diff
@@ -5,8 +5,26 @@ import { Request, Response, NextFunction } from 'express';
 export class RequestLoggerMiddleware implements NestMiddleware {
   private logger = new Logger('HTTP');
 
-  use(request: Request, _response: Response, next: NextFunction): void {
-    this.logger.log(`${request.method} ${request.originalUrl}`);
+  use(request: Request, response: Response, next: NextFunction): void {
+    // const userAgent = request.get('user-agent') || '';
+    // this.logger.verbose(
+    //   `${request.method} ${request.originalUrl} from ${userAgent}`,
+    // );
```

Comment:
Cleanup?

Reply:
The problem here is that when we want to log response status code, we obviously need to wait for that response, and we're not logging anything before that. In case of some request breaks our app completely, with no response, we'll have no logs. This comment code is doing exactly this, it logs request when it starts.
We need to decide what to do with this. I didnt want to create more spam logs, but this one might be needed some day on production. For now, maybe wrap this around `if (development)`?

Reply:
Or just leave it as it is, so we won't forget it when we need it.